src/lib/y2storage/setup_checker.rb

```diff
@@ -138,5 +138,9 @@ def mandatory?(volume)
     def missing?(volume)
       BlkDevice.all(devicegraph).none? { |d| d.match_volume?(volume) }
     end
+
+    def boot_requirements_checker
```

Comment:
NP: I know that it is pretty clear, but please, comment at least the return type.

Reply:
:+1: